lib/streamlit/config.py

```diff
@@ -794,6 +794,22 @@ def _server_address() -> Optional[str]:
     type_=bool,
 )
 
+_create_option(
+    "server.ipHeader",
```

Comment:
Hm, we'll want to check with product on the exact names of these config options + their descriptions (CC @sfc-gh-jcarroll, who mentioned being able to take a look at this PR in the next few days)